packages/main/src/MultiComboBox.ts

```diff
@@ -674,6 +674,7 @@ class MultiComboBox extends UI5Element {
 
 	_onkeydown(e: KeyboardEvent) {
 		const isArrowDownCtrl: boolean = isDownCtrl(e);
+		const isCtrl: boolean = e.metaKey || e.ctrlKey;
```

Comment:
We should check if is it ok this feature to work with both CTRL and CMD combinations on Mac